MAUI/DataGrid/Column-types.md

```diff
@@ -127,7 +127,22 @@ public class DisplayBindingConverter : IValueConverter
 
 In order to set the TextAlignment of the header cell and data row cell , use the [DataGridColumn.HeaderTextAlignment](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_HeaderTextAlignment) and [DataGridColumn.CellTextAlignment](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_CellTextAlignment) property. The default text alignment is based on the type of the columns. The header and data rows are right aligned for numeric, date columns and left aligned for text column.
 
-### Header customizations
+### Line Break Mode
```

Comment:
Modify the title as before like **Header customizations** and move the **Line break mode** topic to the bottom of **Formatting** topic.

The line break mode title should be **Line break mode**